Automattic/jetpack
# Admin UI: update sort order for individual plugins

- Update to assume no position preference and alpha sort those at the same level
- Stats: Modify to use Admin_UI to allow it alpha sorting
- Admin UI: alpha sort features





Fixes #32893

Adds alpha sorting by default for Jetpack menu items, excluding My Jetpack (should be first), Dashboard (flagship plugin, legacy), and Settings (flagship plugin, similar legacy).

## Proposed changes:

* Adds alpha sort layer to the Admin UI package.
* Updates the position values of all plugins to null to allow for alpha sorting, instead of the kinda random status quo.
* Updates the flagship plugin handling of the Stats menu to allow it to alpha sort like the other features.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Install a number of Jetpack plugins. Confirm that "My Jetpack" is first, followed by Dashboard/Settings (if flagship installed), followed by an alpha sorting of the plugins/features installed.
